src/app/modules/registration/components/edit-images/edit-images.component.scss

```diff
@@ -1,23 +1,28 @@
 .images {
-    padding-top: var(--images-padding-top, 0px);
-    padding-bottom: var(--images-padding-bottom, 0px);
-    background: var(--images-background, inherit);
-    .image-wrapper {
-        position: relative;
-        padding-bottom: var(--image-wrapper-padding-bottom, 0px);
-        margin-left: var(--image-wrapper-margin-left, 0px);
-        margin-right: var(--image-wrapper-margin-right, 0px);
-        background: var(--image-wrapper-background, inherit);
+  padding-top: var(--images-padding-top, 0px);
+  padding-bottom: var(--images-padding-bottom, 0px);
+  background: var(--images-background, inherit);
 
-        app-text-comment {
-            display: block;
-            margin-top: -5px;
-            --ion-item-background:  var(--image-comment-background, #fff);
-        }
+  .image-wrapper {
+    position: relative;
+    padding-bottom: var(--image-wrapper-padding-bottom, 0px);
+    margin-left: var(--image-wrapper-margin-left, 0px);
+    margin-right: var(--image-wrapper-margin-right, 0px);
+    background: var(--image-wrapper-background, inherit);
 
-        ion-fab-button {
-            padding: 8px;
-            margin: unset;
-        }
+    app-text-comment {
+      display: block;
+      margin-top: -5px;
+      --ion-item-background: var(--image-comment-background, #fff);
     }
+
+    ion-fab-button {
+      padding: 8px;
+      margin: unset;
+    }
+  }
+}
+
+:host ::ng-deep div.ngx-file-drop__content {
```

Comment:
Er det noen annen måte å fikse dette på? ng-deep er deprecated så vidt jeg vet: https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep